crates/biome_service/src/file_handlers/mod.rs

```diff
@@ -584,6 +601,34 @@ pub(crate) fn parse_lang_from_script_opening_tag(script_opening_tag: &str) -> La
     .map_or(Language::JavaScript, |lang| lang)
 }
 
+pub(crate) fn search_file(
```

Comment:
nit: I would call it `search`, without the use of `file`. While it feels correct now, it won't be once this feature will be wired to additional clients. 

It will be inline with the format capabilities. I suppose we will have `search_range` too in the future.

Reply:
Addressed in https://github.com/biomejs/biome/pull/3288/commits/240455ba282b4061510af5821e1c0bc3ee27ed32.